pom.xml

```diff
@@ -46,27 +61,7 @@
         </configuration>
       </plugin>
 
-      <plugin>
```

Comment:
Imho this should not be removed. If anything it should be updated to java 1.6 or 1.7 if thats what you want to support. But NOT remove. Also it should be in a separate pull request since this is not related to the actual change.